### PR TITLE
[15.0][MIG] web_select_all_companies: Migration to 15.0

### PR DESCRIPTION
**Pull request by BT-mrobles**

### CODE REVIEW

File: web_select_all_companies/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Web Select All Companies",
     "summary": "Allows you to select all companies in one click.",
-    "version": "16.0.1.0.1",
+    "version": "15.0.1.0.1",
```
Comment (BT-anieto) [inline web_select_all_companies/__manifest__.py]: This should be part of the migration commit

**Review by BT-anieto — CHANGES_REQUESTED** [review]
Remember to sqash your commits as stated in https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate

File: web_select_all_companies/__manifest__.py (hunk 1 above)
Comment (BT-mrobles) [reply web_select_all_companies/__manifest__.py]: Fixed @BT-anieto

File: web_select_all_companies/__manifest__.py [hunk 2]
```diff
@@ -4,16 +4,18 @@
...
```
Comment (BT-anieto) [inline web_select_all_companies/__manifest__.py]: This should be 15.0.1.0.0

Comment (BT-mrobles) [reply web_select_all_companies/__manifest__.py]: Updated @BT-anieto

**Review by BT-anieto — APPROVED** [review]
Functional check OK

**Review by OpenCode — APPROVED** [review]

**Pull request merged**